Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/AngularManipulatorCircleViewFeedback.cpp

```diff
@@ -89,6 +91,17 @@ namespace AzToolsFramework
                 manipulatorState.m_localPosition + second * view->m_radius * viewScale);
         }
 
+        debugDisplay.SetColor(AZ::Color(1.f));
```

Comment:
nit: It might be better to move the line color here to an AZ_CVAR to UX can tweak and play with the default color if they want (see the example above for `ed_angularManipulatorCircleFeedbackDisplayColor`)